lang/cs/Org.Apache.REEF.Driver/Task/TaskTransitionState.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.Driver.Task
+{
+    /// <summary>
+    /// Task states in task transitions
+    /// </summary>
+    public enum TaskTransitionState
```

Comment:
DriverTaskState has been used for the class that represents task state and transitions. 
I was using TaskState, but it is easy to cause ambiguous with the Org.Apache.REEF.Common.Runtime.Evaluator.Task.TaskState. May be I just use this name  back as they are in different name spaces and the usages are only in a few classes.